go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.7 // indirect
 	github.com/hashicorp/go-slug v0.16.5
-	github.com/hashicorp/go-tfe v1.78.0
+	github.com/hashicorp/go-tfe v1.78.1-0.20250418170002-da71abb96c5a
```

Comment:
I assume this is just to get nil Description support?

Reply:
Yes, that's correct!